MANIFEST.in

```diff
@@ -30,3 +25,15 @@ prune typings
 prune scripts
 prune hooks
 prune docs
+prune .github
+prune bentoml/testing
```

Comment:
We actually offer some testing functions that users can use, @bojiang can you correct me on this?

Reply:
Those are meant to be used in e2e tests?

Reply:
yes, but then users can also use it as well iirc from my last conversation with bojiang